app/sequencescape/sequencescape/api/v2/specific_tube_creation.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Represents a specific tube creation in Limber via the Sequencescape API
+class Sequencescape::Api::V2::SpecificTubeCreation < Sequencescape::Api::V2::Base
```

Comment:
I don't think I really follow with JSON API why these relationships are defined at all for this endpoint, as the payload is sending some uuids rather than data objects for tubes, parents etc. I guess because it returns the tubes? Maybe you can help me understand how this all fits together.